src/PureCloud.Client/Apis/QualityApi.cs

```diff
@@ -317,4 +317,1202 @@ public async Task<EvaluationResponse> DeleteQualityConversationEvaluationAsync(s
+
+        var uri = UriHelper.GetUri($"api/v2/quality/conversations/{conversationId}/evaluations/{evaluationId}", parameters);
+
+        var response = await client.PutAsJsonAsync(uri, body, _options.JsonSerializerOptions, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<EvaluationResponse>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    public async Task<EvaluationFormResponse> UpdateQualityFormAsync(string formId, EvaluationForm body, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(formId);
+        ArgumentNullException.ThrowIfNull(body);
+
+        var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
+
+        var response = await client.PutAsJsonAsync($"api/v2/quality/forms/{formId}", body, _options.JsonSerializerOptions, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<EvaluationFormResponse>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    public async Task<AiScoringSettings> UpdateQualityFormsEvaluationAiscoringSettingsAsync(string formId, AiScoringSettings body, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(formId);
+        ArgumentNullException.ThrowIfNull(body);
+
+        var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
+
+        var response = await client.PutAsJsonAsync($"api/v2/quality/forms/evaluations/{formId}/aiscoringsettings", body, _options.JsonSerializerOptions, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<AiScoringSettings>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    public async Task<ScorableSurvey> UpdateQualitySurveysScorableAsync(string customerSurveyUrl, ScorableSurvey body, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(customerSurveyUrl);
+        ArgumentNullException.ThrowIfNull(body);
+
+        var parameters = new NameValueCollection();
+        parameters.Add("customerSurveyUrl", UriHelper.ParameterToString(customerSurveyUrl));
+
+        var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
+
+        var uri = UriHelper.GetUri($"api/v2/quality/surveys/scorable", parameters);
+
+        var response = await client.PutAsJsonAsync(uri, body, _options.JsonSerializerOptions, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<ScorableSurvey>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    public async Task<List<Survey>> GetQualityConversationSurveysAsync(string conversationId, CancellationToken cancellationToken = default)
```

Comment:
When returning a collection, always use array since it not mutable. Do the same place where we return a collection like IEnumerable or others